Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -454,8 +475,85 @@ class CredentialsManagerSpec: QuickSpec {
                         }
                     }
                 }
+
             }
-            
+
+            context("forced renew") {
+
+                it("should not yield a new access token without a new scope") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn), scope: "openid profile")
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: nil, minTTL: 0) { error = $0; newCredentials = $1
```

Comment:
let's also do one without specifying param values to ensure defaults are working as expected.

Reply:
Added in https://github.com/auth0/Auth0.swift/pull/399/commits/76f4575cf672ad7792eae20d4c64361a0de4a504.